include/libopencm3/stm32/l0/rcc.h

```diff
@@ -610,6 +640,17 @@ void rcc_osc_ready_int_enable(enum rcc_osc osc);
 void rcc_osc_ready_int_disable(enum rcc_osc osc);
 int rcc_osc_ready_int_flag(enum rcc_osc osc);
 void rcc_wait_for_osc_ready(enum rcc_osc osc);
+enum rcc_osc rcc_get_sysclk(void);
+void rcc_set_sysclk(enum rcc_osc clk);
+void rcc_set_ppre1(uint32_t div);
+void rcc_set_ppre2(uint32_t div);
+void rcc_set_hpre(uint32_t div);
+void rcc_hsi4_clock_setup(void);
+void rcc_hsi16_clock_setup(void);
+void rcc_msi_clock_setup(uint32_t msi_freq);
+void rcc_hse_clock_setup(uint32_t hse_freq);
+void rcc_pll_clock_setup(uint32_t pll_freq, uint32_t src_freq);
+
```

Comment:
The l0 rcc code is incomplete in HEAD so this code is a cherry pick out of the RCC PR to get the necessary function of l0. Once one of these goes to HEAD the other will adapt to use what was decided to go in. Feel free to ignore this code as it will change depending on whether or not the RCC PR is integrated.